### DISCUSSION TITLE
✨ Outline

### DISCUSSION THREAD

**Discussion by JigSawFr** [runtipi/runtipi-appstore#2205] [Requests]
### Description

The fastest knowledge base for growing teams. Beautiful, realtime collaborative, feature packed, and markdown compatible.

### Official Website

https://www.getoutline.com/

### Documentation link

https://docs.getoutline.com/s/guide

### Last application release & date

0.74.0 (6/12/2023)

### Application license

Business Source License 1.1

### Source code repository link

https://github.com/outline/outline

### Docker image link

https://github.com/outline/outline/blob/main/docker-compose.yml

### Please confirm the following

- [X] I confirm that I've put the app name in the title of the discussion
- [X] I confirm that the application is actively maintained.
- [X] I confirm that the docker image of the application is actively maintained.
- [X] I confirm that I have indeed checked that no similar request for this application already exists

### Gathering crowds

Hey folks!

Please upvote 👍 this discussion to show your interest in this request! 

Thanks ⛺

**Comment by JigSawFr** [comment]
Originally requested by @anicholslcsw65

**Comment by steveiliop56** [comment]
Done in #2651
